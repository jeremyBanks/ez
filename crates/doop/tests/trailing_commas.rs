#[test]
fn test() {
    doop::doop! {
        let Numbers = [
            1,
            2,
            3,
            4,
            5,
            6,
        ];
        static {
            let n = 0;
        }
        for N in Numbers {
            let n = n + N;
        }
    }
    assert_eq!(n, 21);
}
