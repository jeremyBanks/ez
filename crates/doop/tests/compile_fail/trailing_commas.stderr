error: expected expression, found `;`
  --> tests/compile_fail/trailing_commas.rs:15:26
   |
15 |             let n = n + N;
   |                          ^ expected expression

error[E0425]: cannot find value `n` in this scope
  --> tests/compile_fail/trailing_commas.rs:18:16
   |
18 |     assert_eq!(n, 21);
   |                ^ not found in this scope
